### ISSUE TITLE
No suitable library found for fods

### ISSUE THREAD

**Issue by retorquere** [pyexcel/pyexcel-odsr#7]
I'm trying to import fods, using this code:

```
#!/usr/bin/env python3

import pandas as pd
from pyexcel_odsr import get_data
import json

path = "/Users/emile/github/better-bibtex/Exported items.fods"
data = get_data(path)
print(json.dumps(data))
```

but I'm getting

```
Traceback (most recent call last):
  File "/Users/emile/Downloads/data_out/data/./load.py", line 9, in <module>
    data = get_data(path)
  File "/usr/local/lib/python3.9/site-packages/pyexcel_odsr/__init__.py", line 43, in get_data
    return read_data(afile, file_type=file_type, **keywords)
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/io.py", line 86, in get_data
    data, _ = _get_data(
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/io.py", line 105, in _get_data
    return load_data(**keywords)
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/io.py", line 191, in load_data
    reader.open(file_name, **keywords)
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/reader.py", line 43, in open
    self.reader_class = NEW_READERS.get_a_plugin(
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/plugins.py", line 185, in get_a_plugin
    plugin = self.load_me_now(f"{location}-{__file_type}", library=library)
  File "/usr/local/lib/python3.9/site-packages/lml/plugin.py", line 292, in load_me_now
    self.raise_exception(key)
  File "/usr/local/lib/python3.9/site-packages/pyexcel_io/plugins.py", line 204, in raise_exception
    raise exceptions.NoSupportingPluginFound(
pyexcel_io.exceptions.NoSupportingPluginFound: No suitable library found for fods
```

**Comment by WardBenjamin** [comment]
+1, I am also having this issue. I'm trying to use this with Sparkfun's CHMT36VA KiCad conversion tool https://github.com/sparkfunX/Desktop-PickAndPlace-CHMT36VA

**Comment by mattmorgan603** [comment]
I think this is fixed with PR #8

**Comment by matkoniecz** [comment]
@chfw it looks like it is fixed now
